test_scripts/API/Restructuring_OnResetTimeout/005_BC_OnResetTimeout_sends_twice_notification.lua

```diff
@@ -0,0 +1,64 @@
+------------------------------------------------------------------------------------------------------------------------
```

Comment:
`005_HMI_sends_BC_OnResetTimeout_notification_twice.lua` might be a better filename for this test

Reply:
@ShobhitAd Please find update in 5cc93b9